docs/docs/api-specification.md

```diff
@@ -100,7 +100,7 @@ load site config -> load plugins -> source nodes -> transform nodes -> create
 graphql schema -> create pages -> compile component queries -> run queries ->
 fin
 
-Once the initial bootstrap is finished, we start `webpack-dev-server` and an express server for serving files for the development server, and for a production build, we start building the CSS then JavaScript then HTML with webpack.
+Once the initial bootstrap is finished a `webpack-dev-server` will be started, and an express server for serving files for the development server, and for a production build, first is starting to build a CSS then JavaScript then HTML with webpack.
```

Comment:
How's this? It makes the sentence a bit more active and adds some detail.
```suggestion
Once the initial bootstrap is finished, a `webpack-dev-server` and express server are started for serving files for the development workflow with live updates. For a production build, Gatsby skips the development server and instead builds the CSS, then JavaScript, then static HTML with webpack.
```

Reply:
thanks for the suggestion, i had commited that :pray: